### PR TITLE
Prtl 598 table sorting

### PR DESCRIPTION
**Pull request by Jephuff**
requires https://github.com/NCI-GDC/gdcapi/pull/706.

### CODE REVIEW

File: modules/node_modules/@ncigdc/routes/AnnotationsRoute.js [hunk 1]
```diff
@@ -5,25 +5,30 @@ import React from 'react';
...
   static prepareParams = ({ location: { search } }) => {
     const q = parse(search);
+    const annotationsSort = q.annotations_sort ? JSURL.parse(q.annotations_sort) : DEFAULT_ANNOTATIONS_SORT;
+
     return ({
       offset: parseIntParam(q.offset, 0),
       size: parseIntParam(q.size, 20),
       filters: parseFilterParam(q.filters, null),
-      annotations_sort: q.annotations_sort || null,
+      annotations_sort: annotationsSort,
```
Comment (shanewilson) [inline modules/node_modules/@ncigdc/routes/AnnotationsRoute.js]: `parseFilterParam(q.sort, null)`?

parseFilter just runs it through `jsurl.parse`

**Comment by shanewilson** [comment]
the sort api works? makes sense to use?

**Comment by Jephuff** [comment]
Yep, I haven't done anything crazy, just passing an array of objects like `[{ field: 'primary_site', order: 'asc' }]` and it looks like it's working. 
I did need to add the `.raw` but I think you warned me about that and it's easy enough to change in the future if the requirement changes.

File: modules/node_modules/@ncigdc/routes/AnnotationsRoute.js (hunk 1 above)
Comment (Jephuff) [inline modules/node_modules/@ncigdc/routes/AnnotationsRoute.js]: ahh makes sense, maybe I should change the name to something like `parseJSURLParam`?

**Pull request merged**